### PR TITLE
Content import/export round-trip tests

### PR DESCRIPTION
**Pull request by jerith**
## Purpose
We currently don't have any real content import/export tests. This PR adds some, primarily to investigate what the existing behaviour is in preparation for refactoring (or rewriting) the content import/export code.

## Approach
By writing tests without touching the code under test (where possible) we'll be able to determine whether the current behaviour is what we intend, and then fix and/or refactor as necessary.

### CODE REVIEW

**Review by rudigiesler — COMMENTED** [review]
Added some comments where I had some context, otherwise looks good to me

File: home/tests/test_content_import_export.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    if not src["translation_tag"]:
+        return src, dst
+    dtags = [tag for tag in dst["tags"].split(", ") if tag != src["translation_tag"]]
+    return src, {**dst, "tags": ", ".join(dtags)}
+
+
+@filter_both
+def ignore_certain_fields(entry):
+    # FIXME: Do we need page.id to be imported? At the moment nothing in the
+    #        import reads that.
```
[the remark this answers is not in the document]
Comment (jerith) [reply home/tests/test_content_import_export.py:40]: In that case, do we even want to export it? (I guess we can document that it's ignored or something.)

File: home/tests/test_content_import_export.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        "media_link",
+        "next_prompt",
+        "related_pages",
+    }
+    return {k: v for k, v in entry.items() if k not in ignored_fields}
+
+
+@filter_both
+def strip_leading_whitespace(entry):
+    # FIXME: Do we expect imported content to have leading spaces removed?
```
[the remark this answers is not in the document]
Comment (jerith) [reply home/tests/test_content_import_export.py:60]: We'll need to document this, then.

**Comment by jerith** [comment]
I'm not sure if the last commit (splitting `home.utils` into `home.content_import_export` and `home.word_embedding`) belongs in here, but I did it because I wanted to look at coverage of the import/export code (which belongs in its own module anyway) and the only things left after moving it were some embedding-related functions and `get_content_depth` which wasn't called anywhere.

**Review by rudigiesler — APPROVED** [review]

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**